### PR TITLE
UX-1449/Tabs Component

### PR DESCRIPTION
**Pull request by jruddell**
image

TODO: update all references to use the new tabs component

### CODE REVIEW

File: src/app/core/components/Navbar.js [hunk 1]
```diff
@@ -28,6 +28,10 @@ import { memoize } from 'utils/misc'
 import generateTestId from 'utils/test-helpers'
 import SimpleLink from './SimpleLink'
 
+/**
+ * @deprecated Use core/elements/sidebar instead
+ */
+
```
Comment (xantorres) [inline src/app/core/components/Navbar.js:34]: can we remove this Navbar if is not being used anymore?

File: src/app/core/elements/tabs/tab.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  useEffect(() => {
+    const { addTab, value, label } = props
+    addTab({ value, label })
+  }, [])
+
+  const { activeTab, value, children } = props
+  if (value !== activeTab) {
+    return null
+  }
+  return <>{children}</>
```
Comment (xantorres) [inline src/app/core/elements/tabs/tab.tsx:19]: You can just `return children` here

**Review by xantorres — APPROVED** [review]
Looks good to me

File: src/app/core/components/Navbar.js (hunk 1 above)
Comment (jruddell) [reply src/app/core/components/Navbar.js:34]: Yes, theres a number of deprecated files that have been marked like this. Will definitely clean them all up

**Pull request merged**